Mailer/app/api/v1/Handlers/Mailers/MailTypesByCodeHandler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Mailers;
+
+use Remp\MailerModule\Api\Validation\InputValidatorTrait;
+use Remp\MailerModule\Repository\ListsRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+
+class MailTypesByCodeHandler extends BaseHandler
+{
+    private $listsRepository;
+    use InputValidatorTrait;
+
+    public function __construct(
+        ListsRepository $listsRepository
+    ) {
+        parent::__construct();
+        $this->listsRepository = $listsRepository;
+    }
+
+    public function handle($payload)
+    {
+        $request = file_get_contents("php://input");
+        $payload = $this->validateInput($request, 'mail-types-by-code.schema.json');
+
+        if ($this->hasErrors()){
+            return $payload;
+        }
+
+        $results = $this->listsRepository->all()->where(['code' => $payload['code']]);
+
+        $output = [];
+        foreach ($results as $row) {
+            $item = new \stdClass();
```

Comment:
Please use arrays (we use arrays everywhere to build responses)

Reply:
Also this part was reused from other file, so I thought it is the right way.

Reply:
I see, that most probably should have been array too :/. Please use arrays from now on.